src/humanPose/HumanPoseAnalyzer.js

```diff
@@ -273,7 +280,9 @@ export class HumanPoseAnalyzer {
      * @param {boolean} historical - whether the pose is historical or live
      */
     poseUpdated(pose, historical) {
-        this.addCloneFromPose(pose, historical);
+        if (this.recordingClones) {
+            this.addCloneFromPose(pose, historical);
```

Comment:
Shouldn't we also add the pose if it's historical, regardless of `recordingClones`? I thought `recordingClones` was meant for live data.

Reply:
That makes sense, I'll adjust accordingly